src/Playwright/Core/Route.cs

```diff
@@ -42,8 +42,9 @@ namespace Microsoft.Playwright.Core;
 internal class Route : ChannelOwner, IRoute
 {
     private readonly RouteInitializer _initializer;
-    private readonly Request _request;
+    internal readonly Request _request;
```

Comment:
Same here. I would expose this using a read-only property.

Reply:
Done.